app/src/main/java/org/jellyfin/androidtv/util/apiclient/StreamHelper.java

```diff
@@ -26,9 +26,33 @@ public static List<MediaStream> getAudioStreams(MediaSourceInfo mediaSource) {
         return getStreams(mediaSource, MediaStreamType.AUDIO);
     }
 
+    public static List<MediaStream> getVideoStreams(MediaSourceInfo mediaSource) {
+        return getStreams(mediaSource, MediaStreamType.VIDEO);
+    }
+
     public static MediaStream getFirstAudioStream(BaseItemDto item) {
-        if (item.getMediaSources() == null || item.getMediaSources().size() < 1) return null;
-        List<MediaStream> streams = getAudioStreams(item.getMediaSources().get(0));
+        return getFirstAudioStream(item, 0);
+    }
+
+    public static MediaStream getFirstAudioStream(BaseItemDto item, int mediaSourceIndex) {
```

Comment:
```suggestion
    public static MediaStream getAudioStream(BaseItemDto item, int mediaSourceIndex) {
```
It's not the first if we explicitly tell it the index.

Reply:
It is the first matching `MediaStream` for the given `MediaSource`.

The idea was to allow the existing functionality of retrieving the first overall media stream, and also an overload allowing to choose a specific source.

Also `getFirstAudioStream` is an existing public util which I was reluctant to change.

Would you rather it be a separate function signature named something like:
`getFirstStreamOfSource(BaseItemDto item, int mediaSourceIndex)`
`getFirstStreamOfSource(BaseItemDto item, int mediaSourceIndex, MediaStreamType streamType)`?

Reply:
This applies to all other comments ofc

Reply:
Aah you're right, I missed that part. This naming is fine 👍.